### DISCUSSION TITLE
(DUVIDA) API Android Studio

### DISCUSSION THREAD

**Discussion by ErikMendesRc** [frontendbr/forum#672] [Discussão]
Boa noites a todos estou com uma dúvida um pouco preocupante para mim a respeito de uma API que desejo utilizar! 
Estou a desenvolver um APP que vai buscar e mostrar as Estatísticas de um perfil de jogador de um determinado jogo! 
Um site oferece uma API 

(Get PUBG Player Stats

GET https://pubgtracker.com/api/profile/pc/{pubg-nickname}

Get PUBG Nickname by SteamID (64-bit number)

GET https://pubgtracker.com/api/search?steamId={STEAM ID}

Authentication
To make use of our APIs we require you to use an API Key. To use the API key you need to pass it along as a header with your requests.

TRN-Api-Key: " Minha chave de acesso aqui" )

Minha dúvida é... Como posso utilizar essa API no Android Studio ?

**Comment by gspaiva** [comment]
Você precisa utilizar alguma API que realiza requisições Http no android. Eu recomendo essa Api -> http://square.github.io/retrofit/ , ela é bem simples e intuitiva.
Você pode aprender por meio da documentação ou por meio de vários videos sobre a mesma que tem na web.
A mesma disponibiliza a opção de fazer requisição utilizando qualquer verbo http (GET,POST,PUT,PATCH,DELETE).
Para cada requisição, você deve adicionar no header da mesma sua api-key (TRN-Api-Key: api-key). É com essa api-key que você terá a permissão para pegar os dados no qual você necessita.

**Comment by ErikMendesRc** [comment]
Isto estaria correto para minha API Key ? 

`public interface RetrofitService {

    @Headers("TRN-Api-Key: 5be0a4c6-6388-4ca6-bf94-a60998fa82a9")`

Será que alguém poderia me ajudar aqui com meu código ?

**Comment by gspaiva** [comment]
@ErikMendesRc sim, estaria.

**Comment by gabrieldarezzo** [comment]
O @viniciusthiengo 

Tem um vídeo sobre consumo de API, bem bacana, veja se ajuda:
https://www.youtube.com/watch?v=L9xcR8SoOZI&t=836s
